README.md

```diff
@@ -41,61 +41,85 @@ You can find a great example of such an issue [here](https://github.com/nodejs/c
 
 ## Meetings
 
-Community Committee meetings will be broadcast via Zoom, will be announced ahead of time for access, and coordinated to optimize for contributor timezones.
+Community Committee meetings are broadcast via Zoom. The join link is published just before the meeting begins in the meeting issue. Meeting times are coordinated to optimize for contributor timezones.
```

Comment:
I'd suggest we change this so that the tool automatically includes the link when the issue is generated.  We can do that if we use a Webinar and the link is the same for every meeting

Reply:
@mhdawson, remind me again where to make that change? 🙂